content/en/docs/reference/command-line-tools-reference/feature-gates/mutating-admission-policy.md

```diff
@@ -0,0 +1,13 @@
+---
+title: MutatingAdmissionPolicy
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: alpha 
+    defaultValue: false
+    fromVersion: "1.30"
+---
+Enables the MutatingAdmissionPolicy in admission chain.
```

Comment:
Eeek. Did we also miss documenting an API?

Reply:
The gate name shows up in all `kube-xyz --help` output. That means it has been exposed to users, no matter it is fully working or it is not working.